lib/public/domain/enums/DetectorTypes.mjs

```diff
@@ -0,0 +1,21 @@
+/**
+ *  @license
+ *  Copyright CERN and copyright holders of ALICE O2. This software is
+ *  distributed under the terms of the GNU General Public License v3 (GPL
+ *  Version 3), copied verbatim in the file "COPYING".
+ *
+ *  See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ *  In applying this license CERN does not waive the privileges and immunities
+ *  granted to it by virtue of its status as an Intergovernmental Organization
+ *  or submit itself to any jurisdiction.
+ */
+
+export const DetectorType = {
+    Physical: 'PHYSICAL',
+    Qc: 'QC',
+    Virtual: 'VIRTUAL',
+    Other: 'OTHER',
```

Comment:
case of RunDefinition (public version) is wrong, for enums properties the conventions is `CAPITAL_SNAKE_CASE` (I will fix that in a PR). Plus, the enum object should be freeze

Reply:
applied